src/pages/ProfilesExplorerView/infrastructure/timeseries/buildTimeSeriesQueryRunner.ts

```diff
@@ -3,7 +3,6 @@ import { SceneQueryRunner } from '@grafana/scenes';
 import { PYROSCOPE_DATA_SOURCE } from '../pyroscope-data-sources';
 import { TimeSeriesQueryRunnerParams } from './TimeSeriesQueryRunnerParams';
 
-// eslint-disable-next-line sonarjs/cognitive-complexity
```

Comment:
Not directly related to this PR but removing an `eslint-disable-next-line`. cannot be bad :)